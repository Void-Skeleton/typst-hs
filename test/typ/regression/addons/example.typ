#let utf8string = read("../something.txt")
