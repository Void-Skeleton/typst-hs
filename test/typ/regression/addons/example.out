--- parse tree ---
[ Code
    "typ/regression/addons/example.typ"
    ( line 1 , column 2 )
    (Let
       (BasicBind (Just (Identifier "test")))
       (FuncExpr
          [ NormalParam (Identifier "x") , NormalParam (Identifier "y") ]
          (Block
             (CodeBlock
                [ If
                    [ ( Equals (Ident (Identifier "x")) (Ident (Identifier "y"))
                      , Block (Content [ Text "\9989" ])
                      )
                    , ( Literal (Boolean True)
                      , Block
                          (Content
                             [ Text "\10060"
                             , Text "("
                             , Code
                                 "typ/regression/addons/example.typ"
                                 ( line 1 , column 47 )
                                 (FuncCall
                                    (Ident (Identifier "repr"))
                                    [ NormalArg (Ident (Identifier "x")) ])
                             , Space
                             , Text "/"
                             , Text "="
                             , Space
                             , Code
                                 "typ/regression/addons/example.typ"
                                 ( line 1 , column 59 )
                                 (FuncCall
                                    (Ident (Identifier "repr"))
                                    [ NormalArg (Ident (Identifier "y")) ])
                             , Text ")"
                             ])
                      )
                    ]
                ]))))
, SoftBreak
, Code
    "typ/regression/addons/example.typ"
    ( line 2 , column 2 )
    (Let
       (BasicBind (Just (Identifier "utf8string")))
       (FuncCall
          (Ident (Identifier "read"))
          [ NormalArg (Literal (String "../something.txt")) ]))
, ParBreak
]
--- evaluated ---
document(body: { text(body: [
]), 
                 parbreak() })
